x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -327,9 +328,26 @@ export function XyToolbar(props: VisualizationToolbarProps<State>) {
               {isFittingEnabled ? (
                 <EuiFormRow
                   display="columnCompressed"
-                  label={i18n.translate('xpack.lens.xyChart.missingValuesLabel', {
-                    defaultMessage: 'Missing values',
-                  })}
+                  label={
+                    <>
+                      {i18n.translate('xpack.lens.xyChart.missingValuesLabel', {
+                        defaultMessage: 'Missing values',
+                      })}{' '}
+                      <EuiIconTip
+                        color="subdued"
+                        content={i18n.translate('xpack.lens.xyChart.missingValuesLabelHelpText', {
+                          defaultMessage: `When there are missing values in an area or line chart these can be represented in different ways in **Lens**. Missing values are drawn using dotted 
+    lines, and will not fill missing values at the left side of the chart.`,
```

Comment:
I think this can be condensed more because the first sentence is pretty vague. How about "Gaps in the data are not shown by default, but can be represented as dotted lines with different modes."